.envrc.example

```diff
@@ -0,0 +1,87 @@
+# .envrc congifuration for jschol
```

Comment:
This file looks like config/env.template except filled in. Where is it used?

Reply:
this is an example direnv configuration, used with direnv